library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
This header file was not intended to include all the js files every time it was called.  The intention was to use it as a js script calling function.  Basically with all these scripts included with this file, it has added all of them to everywhere this function include is used!
  If we want a "include every js library" file (which we might in a great number of places), that should probably be something else, or written as a function within this file, but we should not be automatically including all of them in everything.